packages/tasks/src/tasks/text-classification/about.md

```diff
@@ -150,6 +150,7 @@ classifier("I will walk to home when I went through the bus.")
 
 Would you like to learn more about the topic? Awesome! Here you can find some curated resources that you may find helpful!
 
+- [SetFitABSA: Few-Shot Aspect Based Sentiment Analysis using SetFit](https://huggingface.co/blog/setfit-absa)
```

Comment:
This is a great resource, but maybe a bit advanced to place it the first in the list (not sure)